test/models/content_view_version_test.rb

```diff
@@ -67,6 +69,36 @@ def test_docker_count
       assert_equal image_count, cvv.docker_image_count
       assert_equal tag_count, cvv.docker_tag_count
     end
+
+    def test_search_equal_version
+      assert_includes ContentViewVersion.search_for("version = 1.0"), @cvv
+      query = ContentViewVersion.search_for("version = 1")
+      assert [@cvv, @cvv_minor] & query == [@cvv, @cvv_minor]
```

Comment:
It is not matching the array, I think it is returning more than cvv and cvv_minor, can I check if it returns more than 2 or more results?